src/System.IO.Ports/tests/SerialPort/AbortOnError.cs

```diff
@@ -28,7 +29,12 @@ public class AbortOnError : PortsTest
         /// 3. Verify that the fAbortOnError flag is clear
         /// 
         /// </summary>
-        [ConditionalFact(nameof(HasOneSerialPort))]
+        /// <remarks>
+        /// The tests require access, via reflection, to internal type SerialStream and respective methods GetDcbFlag and
+        /// SetDcbFlag, however, that requires either changes to the public type (incresing its size) or to the test itself.
+        /// Tracking issue at https://github.com/dotnet/corefx/issues/23234.
+        /// </remarks>
+        [ConditionalFact(nameof(HasOneSerialPort), nameof(s_isNotUap))]
```

Comment:
Ah, I wasn't ware that ActiveIssue could be specific to some targets, thanks. I will update this on master.